app/models/embed/purl.rb

```diff
@@ -137,6 +138,30 @@ def initialize(msg = 'The requested PURL resource was not embeddable.')
         super
       end
     end
+
+    class Duration < ISO8601::Duration
+      def to_s
+        raise 'Duration#to_s does not support specifying durations in weeks' if atoms[:weeks] != 0
+        raise 'Duration#to_s does not support specifying negative durations' if atoms.values.detect { |val| val < 0 }
```

Comment:
Yeah, I think logging something.  I'm not sure if there is a way in honeybadger (which we aren't even setup for yet, but no sense in looking into squash) to tell it to log/track an issue that doesn't necessarily raise an error through the actual stack.

But in the meantime, I think logging it would be good and just printing nothing for the duration.